flixel/system/frontEnds/SoundFrontEnd.hx

```diff
@@ -36,6 +36,7 @@ class SoundFrontEnd
 	 * Set this hook to get a callback whenever the volume changes.
 	 * Function should take the form myVolumeHandler(volume:Float).
 	 */
+	@;deprecated("volumeHandler is deprecated, use onVolumeChange, instead")
```

Comment:
there was a typo here, i just pushed a commit to fix it

Reply:
D'oh!